src/style.css

```
@@ -0,0 +1,81 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: stretch;
+}
+
+.block {
+  display: flex;
+  flex-basis: 300px;
+  flex-grow: 1;
+  flex-shrink: 0;
+  height: 300px;
+  font-size: 100px;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+  align-content: stretch;
+}
+
+.block1 {
+  background-color: red;
+}
+
+.block2 {
```

```suggestion
.block:nth-child(2) {
```
Краще використовувати псевдоклас замість того, щоб створювати безліч класів-модифікаторів з цифрами